packages/quality-gates/hooks/quality_gate_config.py

```
@@ -0,0 +1,147 @@
+            common_path = (Path(project_dir) / common_path).resolve()
+        return str(common_path)
+
+    top_level = run_git_command(project_dir, "rev-parse", "--show-toplevel").strip()
+    if top_level:
+        return str(Path(top_level).resolve())
+
+    return str(Path(project_dir).resolve())
+
+
+def _read_state_file(state_file: Path) -> dict:
+    """状態ファイル全体（プロジェクト横断の生データ）を読み込む。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)